### ISSUE TITLE
1080p stream received as 360p stream

### ISSUE THREAD

**Issue by petervanhaaften** [impsnldavid/jit.ndi#10]
**Issue**

A 1080p video stream I have sent via [jit.ndi.send~] is received by [jit.ndi.receive~] as a 360p stream.

Issue has also been confirmed on my side with a stream sent from OBS and received by [jit.ndi.receive~].

![Screen Shot 2020-09-25 at 10 15 27 AM](https://user-images.githubusercontent.com/5347358/94278292-a160e000-ff18-11ea-9ead-55b16591e84f.png)

**Reproduction of issue**

I created a simple max/patch with 1080p colour-bar test video to present my issue. Load patch, click "BANG" above source list, select stream, and (on my side) the 1080p video stream is received as 360p.
 
[pvh-ndi-test.zip](https://github.com/impsnldavid/jit.ndi/files/5283324/pvh-ndi-test.zip)

**Hardware/software specs**

2017 Macbook Pro, running macOS High Sierra 10.13.6, Max/MSP 8.1.5.

**Comment by pixsperdavid** [comment]
Thanks for this, can reproduce on my Macbook Pro 2018, macOS Catalina 10.15.7. Interestingly it's not reproducible on Windows, which explains why I never saw it originally as I do most of my development and testing there.

**Comment by petervanhaaften** [comment]
Thanks for looking into the problem, David. I'm glad it's reproducible on your side, and not some unknown issue with my OSX or NDI setup.

**Comment by pixsperdavid** [comment]
Unfortunately a quick dive into this with the debugger shows that the NDI SDK is actually returning these values for the dimensions of the stream. I'll report this to them but it may not be a quick fix.

**Comment by LSka** [comment]
Hi, I tried the test patch and I'm getting 1920x1080 stream in jit.ndi.receive~
I'm on Max 8.1.6 MacOS 10.14.6, NDI runtime downloaded from here: https://ndi.palakis.fr/runtime/ndi-runtime-4.5.1-macOS.pkg

**Comment by petervanhaaften** [comment]
Hi LSka, thank you very much for your addition to this conversation.

I downloaded the ndi-runtime-4.5.1 package that you suggested. Now the 1080p stream is correctly received by [jit.nid.receive~]! So there was something funny about the package I collected from the NDI Tools site suggested in the git Readme: https://ndi.tv/tools/

![Screen Shot 2020-09-25 at 12 23 34 PM](https://user-images.githubusercontent.com/5347358/94291946-4801ac80-ff2a-11ea-8d37-1049339ddda2.png)

I'm very happy to have this package working. Thanks again David for your work on this project.
